src/libraries/System.Private.CoreLib/src/System/ArrayTypeMismatchException.cs

```diff
@@ -1,23 +1,15 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-/*=============================================================================
-**
-**
-**
-** Purpose: The arrays are of different primitive types.
-**
-**
-=============================================================================*/
-
 using System.ComponentModel;
 using System.Runtime.CompilerServices;
 using System.Runtime.Serialization;
 
 namespace System
 {
-    // The ArrayMismatchException is thrown when an attempt to store
-    // an object of the wrong type within an array occurs.
+    /// <summary>
+    /// The exception that is thrown when there is an attempt to read or write protected memory.
+    /// </summary>
```

Comment:
Also a mismatch here. Specific for array handling

Reply:
Would you believe my keyboard is having trouble with Ctrl-C...!